eacl/table.go

```diff
@@ -155,10 +155,7 @@ func NewTableFromV2(table *v2acl.Table) *Table {
 			t.cid = new(cid.ID)
 		}
 
-		var h [sha256.Size]byte
-
-		copy(h[:], id.GetValue())
-		t.cid.SetSHA256(h)
+		t.cid.SetSHA256([sha256.Size]byte(id.GetValue()))
```

Comment:
if a cid was 31 bytes long, it did not panic before but now it will. however, it does not seem to be a behaviour regression but panicing in SDK can be bad

Reply:
yeah, i intentionally posted it like this on the review. To me, panic on bug is better than silent bug coverage. The bugfix requires function signature change, but i dont see any other option

ur thoughts?

Reply:
we may also add regular `ReadFromV2` method returning error and deprecated this func